addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18880,7 +18880,19 @@ msgctxt "#36618"
 msgid "Add-ons will be given access to personal data stored on this device. By allowing, you agree that you are solely responsible for any loss of data, unwanted behaviour, or damage to your device. Proceed?"
 msgstr ""
 
-#empty strings from id 36619 to 36899
+#. Label of a setting, allow the user to specifiy high quality downscaling of pictures
+#: system/settings/settings.xml
+msgctxt "#36619"
+msgid "High quality downscaling"
+msgstr ""
+
+#. Description of setting "Pictures -> Slideshow -> High Quality Downscaling" with label #36619
+#: system/settings/settings.xml
+msgctxt "#36620"
+msgid "Enable high quality downscaling of pictures."
```

Comment:
@FernetMenta remember, users are usually dumb, just like customers in your day job (at least in mine). I doubt anyone is thinking about any consequences or impact when happily enabling random settings, so you better explain everything in the settings description like you would to a little child to prevent a flood of support requests on the forum etc